.bazelrc

```diff
@@ -12,6 +12,7 @@ build \
   --verbose_failures \
   --workspace_status_command=envoy/bazel/get_workspace_status \
   --xcode_version=11.0.0 \
+  --incompatible_depset_is_not_iterable=false \
```

Comment:
This is because I get this error: ```ERROR: /Users/runner/runners/2.158.0/work/envoy-mobile/envoy-mobile/library/kotlin/src/io/envoyproxy/envoymobile/BUILD:6:1: in pom_file rule //library/kotlin/src/io/envoyproxy/envoymobile:android_aar_pom: 
Traceback (most recent call last):
	File "/Users/runner/runners/2.158.0/work/envoy-mobile/envoy-mobile/library/kotlin/src/io/envoyproxy/envoymobile/BUILD", line 6
		pom_file(name = 'android_aar_pom')
	File "/private/var/tmp/_bazel_runner/0e7f6719929f1ce7da05a83d96cc0859/external/google_bazel_common/tools/maven/pom_file.bzl", line 151, in _pom_file
		_sort_artifacts(mvn_deps, ctx.attr.preferred_group...)
	File "/private/var/tmp/_bazel_runner/0e7f6719929f1ce7da05a83d96cc0859/external/google_bazel_common/tools/maven/pom_file.bzl", line 120, in _sort_artifacts
		for artifact in artifacts: ...
type 'depset' is not iterable. Use the `to_list()` method to get a list. Use --incompatible_depset_is_not_iterable=false to temporarily disable this check.
ERROR: Analysis of target '//:android_dist' failed; build aborted: Analysis of target '//library/kotlin/src/io/envoyproxy/envoymobile:android_aar_pom' failed; build aborted```

Reply:
We should try and update that repo, they've likely fixed it

Reply:
I don't actually know how to map their artifact to a release/branch (they don't have a release). The `pom_file.bzl` hasn't been updated to use something that's not deps: https://github.com/google/bazel-common/blob/master/tools/maven/pom_file.bzl#L151